src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/QueueConfig.java

```diff
@@ -36,7 +42,23 @@ public void initialise() throws ServiceBusException, InterruptedException {
         ExecutorService executorService = Executors.newSingleThreadExecutor(
             namedThreadFactory
         );
-        client.registerMessageHandler(messageHandler, executorService);
+
+        // Note: lazy init with retry otherwise AKS setup fails as a queue is
+        // created in that environment only after deployment is complete.
+        int tries = 0;
+        while (true) {
+            try {
+                client.registerMessageHandler(messageHandler, executorService);
+                return;
+            } catch (Throwable t) {
```

Comment:
is there not a more specific error we can catch?

Reply:
It's all very exploratory at the moment. I've added a WIP label. Not sure if this can be done at all but it's needed for AKS to work

Reply:
Yes, it can be done and catching a more specific error.